### PR TITLE
add `od_from_atlas_group`, `od_from_p_subgroup`, and helpers

### PR DESCRIPTION
**Pull request by ThomasBreuer**
- `od_from_atlas_group(chi)` tries to find a suitable representation affording `chi` in the Atlas of Group Representations, and then to compute the orth. disc. from it
- `od_from_p_subgroup(chi, p)` tries to use the restriction of `chi` to the Sylow `p`-subgroup
- add `known_class_fusions`
- add optional argument to `induced_cyclic`
- add helper `possible_permutation_characters_from_sylow_subgroup`
- `orbits_group_automorphisms` is used to reduce the tests to only one member of each orbit on irreducible characters
- the new optional `min_char` argument in `isomorphic_group_over_finite_field` is used to guarantee that the reduction of an irreducible module is irreducible
- the other changes are minor improvements

(This branch tries to copy the relevant commits from the corrupted branch  #2960.)

### CODE REVIEW

File: experimental/OrthogonalDiscriminants/src/direct.jl [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    end
+
+    if ! haskey(info, :constituents)
+      # We do not yet know that `G` has character `chi`.
+      # Check whether the representation is abs. irred.
+      if p == 0
+        # We can check abs. irred. if we can reduce the repres.
+        # modulo a prime l that does not divide the group order.
+        l = next_prime(max([x[1] for x in factor(order(tbl))]...))
+        Gbar, _ = Oscar.isomorphic_group_over_finite_field(G, min_char = Int(l))
```
Comment (fieker) [inline experimental/OrthogonalDiscriminants/src/direct.jl:195]: Ideally this should be 
```M = gmodule(Gbar)
is_abs_irr(M)
```
or similar. The "natural" gamodule of a matrix group is the missing bit...

File: experimental/OrthogonalDiscriminants/src/theoretical.jl [hunk 2]
```diff
@@ -172,3 +175,153 @@ function od_for_specht_module(chi::GAPGroupClassFunction)
...
+  if cand != nothing
+    res = collect(Set([od_from_p_subgroup(chi, p, pi) for pi in cand]))
+    length(res) == 1 && return res[1]
+  end
+  return false, ""
+end
+
+function od_from_p_subgroup(chi::GAPGroupClassFunction, p::Int,
+                            pi::GAPGroupClassFunction)
+  tbl = chi.table
```
Comment (fieker) [inline experimental/OrthogonalDiscriminants/src/theoretical.jl:224]: parent(chi) or character_table(chi)?

File: src/Rings/AbelianClosure.jl [hunk 3]
```diff
@@ -459,6 +469,7 @@ function minimize(::typeof(CyclotomicField), a::nf_elem)
   return minimize(CyclotomicField, [a])[1]
 end
 
+#TODO: document this!
```
Comment (fieker) [inline src/Rings/AbelianClosure.jl:472]: this function is ambigous: the element defines an order - which has a conductor as an order, and a field. The notions are different. So a comment/ doc would be important

**Review by fieker — APPROVED** [review]

**Comment by ThomasBreuer** [comment]
@fieker Thanks for the feedback. I am going to address your comments.
(No need to ask for not merging, since some required tests fail.)

**Pull request closed without merging**